src/main/java/com/biel/lobby/GestorMapes.java

```diff
@@ -71,31 +71,47 @@ public void queryAutoRatings() {
 		auto_ratings = Com.getDataAPI().getAutoRating();
 		Mapes.sort((m1, m2) -> Double.compare(m2.getRating(), m1.getRating()));
 	}
-	public void ObrirMenuMapes(Player ply){
+	public void ObrirMenuMapes(Player ply) {
 		queryAutoRatings();
-		IconMenu menu = new IconMenu(ChatColor.RED + "Tots els mapes", (int) (9 * (Math.ceil(Mapes.size() / 9) + 1)), event -> {
+
+		String menuTitle = ChatColor.RED + "Tots els mapes";
```

Comment:
You cannot do that since this causes a bug where the menu instance is shared by all users. I don't append random stuff to things just for fun!